docs/review-queries.md

```
@@ -0,0 +1,69 @@
+
+### Daily P&L
+
+```
+trade_event_type = "exit"
+(group by) date(timestamp)
+(sum) realized_pnl
+```
+
+### Win rate by strategy
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)